README.md

```diff
@@ -109,6 +130,7 @@ Pegasystems uses a standard naming practice of hostname/product/image:tag. Pega
 `platform/installer`   | A utility image with which you install all of the Pega-specific rules and database tables in the “Pega” database that you have configured for your deployment. This installation is required before a deployment can take place.| `<version>` |
 `platform/pega`  | (Download required) Deploys Pega Platform with its customized version of the Tomcat application server.| `<version>` or `<version>-YYYYMMDD` |
 `platform/search` | (Download required) Deploys the required search engine for Pega Platform search and reporting capabilities. This Docker image contains Elasticsearch and includes all required plugins.| `<version>` or `<version>-YYYYMMDD` |
+`platform/srs` | (Download required) Deploys the required search engine for Pega Infinity next-gen search and reporting capabilities. This Docker image contains Pega Search and Reporting Service.| `<version>` or `<version>-YYYYMMDD` |
```

Comment:
Might want to note here that this is for 8.6+, and used instead of platform/search.

Reply:
Is this a requirement for all 8.6+ systems so pega/search is deprecated? Or can existing clients on a previous version still use pega/search? I think that the comments in the values.yaml file also has to be updated for 8.6+ to reflect this option or deprecation.

Reply:
I can add below comment to the "pegasearch" section in pega/values.yaml 
"From Pega 8.6+ onwards, instead of `pegasearch` based internal search setup, pega suggests use the chart 'backingservices/srs' to deploy a shareable `Search and Reporting Service` deployment, which is shareable and independently manageable search solution for the pega platform."

Reply:
I think we're a bit off in this statement. The Docker image is named 'platform/srs', not 'backingservices/srs' according to the README edits. Also, can we tweak the wording a bit so it complies with techdoc standards?

"For Pega Platform 8.6 and later, Pega recommends using the chart 'backingservices/srs' to enable the use of a search and reporting service in your deployment. This is a shareable and independently manageable search solution for Pega Platform what replaces the previously used elastic search. To use this search and reporting service, you must specify the use of the Pega-provided docker image, `platform/srs`, for the pegasearch.image: parameter.

Can you also updated the pega/values.yaml file comments here appropriately? I'm not sure if they should be replaced or updated. Thanks!!
# Elasticsearch deployment settings.
# Note: This Elasticsearch deployment is used for Pega search, and is not the same Elasticsearch deployment used by the EFK stack.
# These search nodes will be deployed regardless of the Elasticsearch configuration above.

+ onwards, instead of `pegasearch` based internal search setup, p
`Search and Reporting Service` deployment, which is a shareable and independently manageable search solution for the pega platform."

Reply:
sorry a little correction there " To use this search and reporting service, you must specify the use of the Pega-provided docker image, platform/srs, for the pegasearch.image: parameter".

The new subchart "srs" defined under chart "backingservices" should be deployed before pega deployment and the service url be configured in pega chart values using the "externalURL" parameter. When "externalURL" parameter is populated, we conditionally do not deploy the pegasearch subchart resources as platform will use the srs service provided at the "externalURL" for search functionality.

Reply:
@taz-pega-work I have added a new commit with the documentation/comment changes as suggested, please review

Reply:
You shouldn't describe what is not happening; instead discuss what does occur. How’s this?

"To use this search and reporting service, you must specify the use of the Pega-provided docker image, platform/srs, for the pegasearch.image: parameter".

The new subchart "srs" defined under chart "backingservices" must be deployed before pega deployment and the service url must be configured in pega chart values using the ".Values.pegasearch.externalURL" parameter. When you configure the ".Values.pegasearch.externalURL" parameter, your deployment uses the srs service provided at the ".Values.pegasearch.externalURL" for Pega Platform search functionality. The deployment uses the parameter configuration in .Values.pegasearch.externalSearchService to indicate pega to connect to external srs.”

I would also state whether Elasticsearch is still supported, and if it is, say what is the expected default. Thanks for reviewing the details!